src/worker.js

```diff
@@ -92,6 +92,7 @@ const initServer = (app, config, callback) => {
   runAppLifecycle('initialize', config.plugins, config)
     .then(() => {
       server = app.listen(config.port, callback);
+      return null;
```

Comment:
Is the issue here that `listen` returns a promise? Why not just return that promise instead? (Same with all of them)